src/main/scala/beam/agentsim/agents/ridehail/RideHailManager.scala

```diff
@@ -1364,13 +1364,15 @@ class RideHailManager(
     tick: Int,
     finalTriggersToSchedule: Vector[ScheduleTrigger]
   ): Unit = {
-    log.debug(
-      "Removing request: {} pendingAcks: {} pendingRoutes: {} requestBufferSize: {}",
-      requestId,
-      s"(${pendingModifyPassengerScheduleAcks.size}) ${pendingModifyPassengerScheduleAcks.keySet.map(_.toString).mkString(",")}",
-      numPendingRoutingRequestsForReservations,
-      rideHailResourceAllocationManager.getBufferSize
-    )
+    if (log.isDebugEnabled) {
```

Comment:
This is duplicated from the log itself- https://github.com/akka/akka/blob/master/akka-actor/src/main/scala/akka/event/Logging.scala#L1378 

Unless it is primarily focused on the building up of pendingAcks and other args?

Reply:
I figured